reservations/allocation_models.py

```diff
@@ -0,0 +1,126 @@
+import datetime
+
+from django.utils import timezone
+
+from applications.models import Application, ApplicationEvent
+from reservation_units.models import ReservationUnit
+
+
+class AvailableTime(object):
+    def __init__(self, start: int, end: int):
+        self.start = start
+        self.end = end
+
+
+# Used to control precision of allocation. Currently we allocate at 15 minute precision
+ALLOCATION_PRECISION = 15
+
+
+class AllocationSpace(object):
+    def __init__(
+        self,
+        id: int,
+        period_start: datetime.date,
+        period_end: datetime.date,
+        times: [AvailableTime],
+    ):
+        self.id = id
+        self._period_start = period_start
+        self._period_end = period_end
+        self.available_times: [AvailableTime] = times
+
+    def addTime(self, start: datetime, end: datetime):
```

Comment:
:camel: :eyes:

Reply:
But camel case is superior... Changing python conventions one method at a time.